src/asm/statements.rs

```diff
@@ -174,7 +187,7 @@ impl<'a> Label<'a> {
         map(
             terminated(take_while1(good_for_label), tag(":")),
             |id: &str| {
-                let local = id.starts_with(".L");
+                let local = id.starts_with(".L") || id.starts_with("LBB") || id.starts_with("Ltmp");
```

Comment:
Does it make sense to use  the `LOCAL_LABELS` regexp here now that we know there's all sorts of strange labels?

Reply:
That does sound like a good idea, and it probably wouldn't change the perf that much.